### PR TITLE
Toggle optin

### LINKED ISSUE #13: User Story 1: Profile Page Checkbox
As a registered user,
When I go to my Profile page, 
I can click the Misc submenu,
then I see a checkbox for “Receive your Daily Dev”.

### LINKED ISSUE #14: User Story 2: Daily Dev Sign-up
As a user, when I see the checkbox on the profile page for Daily Dev Email, 
I see a field next to it that pre-populates my email address,
and a button that allows me to save my preferences.

When I check the box, submit an email, and save my preferences, 
I see a flash message letting me know I am now subscribed to the Daily Dev email, 
And this subscription is saved in the database.

### LINKED ISSUE #17: User Story 3: Daily Dev Email Send
- [ ] Create rake task to send email
- [x] Create mailer & worker
- [x] Setup mailcatcher
- [x] Send email to specified user

### PR DESCRIPTION
**Pull request by msimon42**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [x] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description
Adds method to user model that sends daily dev email to all users opted in. Adds background worker for sending emails. 
## Related Tickets & Documents
closes #17 
closes #14 
closes #13 

## Added tests?

- [x] yes
- [ ] no, because they aren't needed
- [ ] no, because I need help

## Added to documentation?

- [ ] docs.dev.to
- [ ] readme
- [x] no documentation needed

## [optional] What gif best describes this PR or how it makes you feel?
💩 
alt_text

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -86,6 +86,7 @@ gem "rubyzip", "~> 2.1" # Rubyzip is a ruby library for reading and writing zip
 gem "s3_direct_upload", "~> 0.1" # Direct Upload to Amazon S3
 gem "sass-rails", "~> 6.0" # Sass adapter for the Rails asset pipeline
 gem "sidekiq", "~> 6.0.4" # Sidekiq is used to process background jobs with the help of Redis
+gem "sinatra"
 gem "sitemap_generator", "~> 6.1" # SitemapGenerator is a framework-agnostic XML Sitemap generator
```
Comment (mel-rob) [inline Gemfile:90]: What'd you have to add Sinatra for?

**Review by mel-rob — APPROVED** [review]
Looks great. Thank you for figuring out that email issue!

File: config/environments/development.rb [hunk 2]
```diff
@@ -75,13 +75,14 @@
   config.action_mailer.perform_deliveries = true
   config.action_mailer.default_url_options = { host: config.app_domain }
   config.action_mailer.smtp_settings = {
-    address: "smtp.gmail.com",
```
Comment (grayson-palmer) [inline config/environments/development.rb:78]: I'm guessing this was due to it not being a deployed live site, but this allows it to  function locally? Nice job working that out!

**Review by grayson-palmer — APPROVED** [review]
Nice job with all of that! I think I get for the most part how that all came together. Great work!

**Pull request merged**